docs/src/main/sphinx/functions/json.rst

```diff
@@ -2,6 +2,1174 @@
+''''''''''''''''''''''''''''
+
+.. code-block:: text
+
+    + <path>
+    - <path>
+
+The operand ``<path>`` is evaluated to a sequence of items. It is required that
+every item is a numeric value. The unary plus or minus is applied to every item
+in the sequence, following SQL semantics, and the results form the returned
+sequence.
+
+The remaining JSON path operations are collectively known as **accessors**.
+
+member accessor
+'''''''''''''''
+
+The member accessor returns the value of the member with the specified key for
+each JSON object in the input sequence.
+
+.. code-block:: text
+
+    <path>.key
+    <path>."key"
+
+The condition when a JSON object does not have such a member is called a
+structural error. In the lax mode, it is suppressed, and the faulty object is
+excluded from the result.
+
+Let ``<path>`` return a sequence of three JSON objects:
+
+.. code-block:: text
+
+    {"customer" : 100, "region" : "AFRICA"},
+    {"region" : "ASIA"},
+    {"customer" : 300, "region" : "AFRICA", "comment" : null}
+
+the expression ``<path>.customer`` succeeds in the first and the third object,
+but the second object lacks the required member. In strict mode, path
+evaluation fails. In lax mode, the second object is silently skipped, and the
+resulting sequence is ``100, 300``.
+
+It is required that all items in the input sequence are JSON objects.
+
+.. note::
+
+    Trino implementation does not support JSON objects with duplicate keys.
+
+wildcard member accessor
+''''''''''''''''''''''''
+
+Returns all bound values for each JSON object in the input sequence. All the
+partial results are concatenated into the returned sequence.
+
+.. code-block:: text
+
+    <path>.*
+
+Let ``<path>`` return a sequence of three JSON objects:
+
+.. code-block:: text
+
+    {"customer" : 100, "region" : "AFRICA"},
+    {"region" : "ASIA"},
+    {"customer" : 300, "region" : "AFRICA", "comment" : null}
+
+The results is:
+
+.. code-block:: text
+
+    100, "AFRICA", "ASIA", 300, "AFRICA", null
+
+It is required that all items in the input sequence are JSON objects.
+
+The order of the returned values follows the order of the original JSON
+objects. However, within objects, the order is arbitrary.
+
+array accessor
+''''''''''''''
+
+Returns the elements at the specified indexes for each JSON array in the input
+sequence. Indexes are zero-based.
+
+.. code-block:: text
+
+    <path>[ <subscripts> ]
+
+The ``<subscripts>`` list contains one or more subscripts. Each subscript
+specifies a single index or a range (ends inclusive):
+
+.. code-block:: text
+
+    <path>[<path1>, <path2> to <path3>, <path4>,...]
+
+After the input sequence is evaluated, in the lax mode, all non-array items
+in the sequence are wrapped in singleton arrays. Array wrapping replaces the
+usual unwrapping, because the array accessor requires that all the input items
+are JSON arrays.
+
+For every array in the input sequence, all subscript indexes are computed, with
```

Comment:
Is this relevant? It seems kind of obvious that that would happen: "For every array in the input sequence, all subscript indexes are computed". Also, the passive voice makes it sound awkward.

Reply:
changed